### ISSUE TITLE
Roles for Users

### ISSUE THREAD

**Issue by DruzhininPavel** [EpamLifeSciencesTeam/random_coffee#11]
You need to add roles for users. It is desirable that roles can be added or removed. User roles must be an array. those. the user can be both User and Administrator and Deleted, etc. Based on this, the user may have different access to different sections of the system.

**Comment by myazinn** [comment]
What do you mean by the "Deleted" role?

**Comment by DruzhininPavel** [comment]
> What do you mean by the "Deleted" role?

I thought, that we can set deleted status for user by the role. It s like pending status but instead. If user role was setting as deleted, that system can remove it after week or month:) But, we need to discuss it. :)))

**Comment by myazinn** [comment]
That doesn't sound as a role to me, rather an "account status".

**Comment by DruzhininPavel** [comment]
Yes, Shure. You are right. That we should create another task, with user status field.

**Comment by DruzhininPavel** [comment]
Yes, Shure. You are right. That we should create another task, with user status field.
